<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="Button_time">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="Button_Lp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>(</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="Button_Rp">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="Button_CE">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>CE</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="Button_C">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="Button_7">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Button_8">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="Button_9">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="Button_Div">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="Button_4">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="Button_5">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="Button_6">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="Button_Mul">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="Button_1">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="Button_2">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="Button_3">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="Button_Minus">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="Button_0">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="Button_point">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="Button_equal">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="Button_Plus">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
